services/enqueueing/enqueueing.go

```diff
@@ -0,0 +1,45 @@
+package enqueueing
+
+import (
+	"errors"
+
+	"google-scraper/database"
+	"google-scraper/models"
+
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/gocraft/work"
+	"github.com/beego/beego/v2/server/web"
+)
+
+var enqueuer *work.Enqueuer
```

Comment:
In the `init()` function I am initializing the new enqueuer with redis pool and assign it to `enqueuer` variable so that it does not call/execute everytime I call `EnqueueKeywordJob` function. Then access it from the `EnqueueKeywordJob` function. Thats why I need to define it.